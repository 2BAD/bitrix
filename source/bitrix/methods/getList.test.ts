/* eslint-env jest */
// tslint:disable: no-expression-statement

describe('Bitrix `handleGetListPayload` method', () => {
  it.todo('should return payload')
  it.todo('should throw when getting errors in payload')
})

describe('Bitrix `getList` method', () => {
  it.todo('should form a proper request')
  it.todo('should return body as payload')
  it.todo('should cast payload to the <P>')
  it.todo('should throw when getting errors in payload')
})
