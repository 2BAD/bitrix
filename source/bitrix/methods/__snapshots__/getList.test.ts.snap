// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bitrix \`getList\` method should throw when getting errors in payload 1`] = `[Error: [get] failed to get the resource: Expected error for \`getList\`.]`;

exports[`Bitrix \`handleGetListPayload\` method should throw when getting errors in payload 1`] = `"[get] failed to get the resource: Expected error for \`handleGetListPayload\`."`;
